terraform/aptos-node-testnet/main.tf

```diff
@@ -56,7 +56,7 @@ module "aptos-node-helm-values-deepmerge" {
   # https://registry.terraform.io/modules/Invicton-Labs/deepmerge/null/0.1.5
   source = "Invicton-Labs/deepmerge/null"
   maps = [
-    var.enable_forge ? local.aptos_node_helm_values_forge_override : map(),
+    var.enable_forge ? tomap(local.aptos_node_helm_values_forge_override) : {},
```

Comment:
oh it's because we have this: https://github.com/aptos-labs/aptos-core/blob/main/terraform/aptos-node/aws/variables.tf#L67

Shall we change to `map`? https://www.terraform.io/language/expressions/types#types